src/MonitoringFunctions/Properties/serviceDependencies.json

```diff
@@ -0,0 +1,12 @@
+{
+  "dependencies": {
+    "secrets1": {
+      "type": "secrets",
+      "connectionId": "monitoring-functions-push-connection-string"
+    },
+    "storage1": {
+      "type": "storage",
+      "connectionId": "AzureWebJobsStorage"
+    }
+  }
+}
```

Comment:
white space

Reply:
Must be placed by github. There is no whitespace in the file.